File: Easy/LCE - 7.js
/**
 * @param {number} x
 * @return {number}
 */
var reverse = function(x) {
    
    const negative = x<0 ? -1 : 1
    let output = 0
    x = Math.abs(x)
    
    while(x>0) {
        let temp = x%10
        output = output*10 + temp
        x = Math.floor(x/10)
    }
    
    if(output>Math.pow(2,31)) return 0
    return output*negative
};

File: Easy/Python - Solutions/LCE_7.py
class Solution:
    def reverse(self, x: int) -> int:
        if x < 0:
            sign = -1
            x = x*-1
        else:
            sign = 1

        sums = 0
        while x > 0:
            temp = x % 10
            sums = sums*10 + temp
            x //= 10

        if not -2**31 <= sums <= 2**31-1:
            return 0

        if sign == -1:
            return sums*-1
        return sums
